lib/cn/kvset_split.h

```diff
@@ -0,0 +1,47 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/*
+ * Copyright (C) 2022 Micron Technology, Inc.  All rights reserved.
+ */
+
+#ifndef HSE_KVS_CN_KVSET_SPLIT_H
+#define HSE_KVS_CN_KVSET_SPLIT_H
+
+#include <hse_util/hse_err.h>
+#include <hse_util/inttypes.h>
+#include <hse_util/hlog.h>
+#include <hse_util/key_util.h>
+
+#include "kblock_reader.h"
+#include "wbt_reader.h"
+#include "blk_list.h"
+#include "cn_tree.h"
+#include "hblock_reader.h"
+#include "hblock_builder.h"
+
+/**
+ * struct kvset_split_res - split output
+ */
+struct kvset_split_res {
+    struct kvset_mblocks     blks_left;   /* kblocks, vblocks and the hblock for the left kvset */
```

Comment:
Same idea with my array comment previously

Reply:
Changed this to array of kvset_mblocks.